kubernetes/deployment-production.tmpl

```diff
@@ -36,4 +36,29 @@ spec:
     - protocol: TCP
       port: 80
       targetPort: 80
-  type: NodePort
\ No newline at end of file
+  type: NodePort
+
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: zoomapper-production-ingress
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    certmanager.k8s.io/cluster-issuer: letsencrypt-prod
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+    nginx.ingress.kubernetes.io/proxy-buffer-size: "128k"
+    nginx.ingress.kubernetes.io/set-real-ip-from: "10.0.0.0/8"
+spec:
+  tls:
+  - hosts:
+    - zoomapper.zooniverse.org
+    secretName: zoomapper-production-tls-secret
+  rules:
+  - host: zoomapper.zooniverse.org
+    http:
+      paths:
+      - backend:
+          serviceName: zoomapper-production-app
+          servicePort: 80
+        path: /(.*)
```

Comment:
Remove the regex match for the same reasons as https://github.com/zooniverse/static/pull/176
```suggestion
        path: /

```